src/shapes/object.class.js

```diff
@@ -573,6 +573,14 @@
     selectable:               true,
 
     /**
+     * When set to `false`, an object can not be selected for modification using group-based selection.
+     * But events still fire on it.
+     * @type Boolean
```

Comment:
Sorry, I incorrectly commited that comment, it has no meaning. It is now removed.